api_tests/tokens/test_views.py

```diff
@@ -0,0 +1,277 @@
+import copy
+import mock
+
+from nose.tools import *  # flake8: noqa
+
+from website.models import User, ApiOAuth2PersonalToken
+from website.util import api_v2_url
+from website.util import sanitize
+
+from tests.base import ApiTestCase
+from tests.factories import ApiOAuth2PersonalTokenFactory, AuthUserFactory
+
+
+def _get_token_detail_route(token):
+    path = "tokens/{}/".format(token._id)
+    return api_v2_url(path, base_route='/')
+
+
+def _get_token_list_url():
```

Comment:
Probably don't need a helper method for this, since it always returns the same value.

(the existence of a similar method in applications tests is largely historical crud from before we simplified that route)